storage/buckets/get_bucket_metadata.go

```diff
@@ -1,4 +1,4 @@
-// Copyright 2019 Google LLC
```

Comment:
Don't update copyright years for existing files. New files should be 2020, though.

Reply:
Done